lib/cyber_auth_module_integrator/core/core.rb

```diff
@@ -0,0 +1,59 @@
+require 'json'
+require 'ostruct'
+
+module CyberAuthModuleIntegrator
+  # TODO
+  class Core
+    def initialize
+      @entity = EntityProcessor.new
+    end
+
+    def enrichment(message)
+      @message = operation_type(json_to_object(message))
+      entity_type(@message)
```

Comment:
Lo mismo que para operation_type y por simetría, si el otro método retorna el message modificado, este también tendría que hacerlo, y hacer innecesario el @message posterior. O buscar la simetría al revés, que ninguno de los dos devuelva el mensaje modificado, algo así como
```suggestion
@message = json_to_object(message)
calculate_operation_type(@message)
calculate_entity_type(@message)
@message
```

Reply:
Fixed!